docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -664,6 +670,246 @@ Delegates votes from the sender to `delegatee`.
 
 [.contract-item]
 [[IVotes-delegate_by_sig]]
-==== `[.contract-item-name]#++delegate_by_sig++#++(delegator: ContractAddress, delegatee: ContractAddress, nonce: felt252, expiry: u64, signature: Array<felt252>)++` [.item-kind]#external#
+==== `[.contract-item-name]#++delegate_by_sig++#++(delegator: ContractAddress, delegatee: ContractAddress, nonce: felt252, expiry: u64, signature: Span<felt252>)++` [.item-kind]#external#
 
 Delegates votes from `delegator` to `delegatee`.
+
+[.contract]
+[[VotesComponent]]
+=== `++VotesComponent++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.17.0/packages/governance/src/votes/votes.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_governance::votes::VotesComponent;
+```
+Component that implements the {IVotes} interface and provides a flexible system for tracking and delegating voting power.
+
+By default, token balance does not account for voting power. This makes transfers cheaper. The downside is that it requires users to delegate to themselves in order to activate checkpoints and have their voting power tracked.
+
+NOTE: When using this module, your contract must implement the {VotingUnitsTrait}. For convenience, this is done automatically for `ERC20` and `ERC721` tokens.
+
+[.contract-index]
+.Voting Units Trait Implementations
+--
+.ERC20VotesImpl
+* xref:#VotesComponent-ERC20VotesImpl-get_voting_units[`++get_voting_units(self, account)++`]
+
+.ERC721VotesImpl
+* xref:#VotesComponent-ERC721VotesImpl-get_voting_units[`++get_voting_units(self, account)++`]
+--
+
+[.contract-index#VotesComponent-Embeddable-Impls]
+.Embeddable Implementations
+--
+[.sub-index#VotesComponent-Embeddable-Impls-VotesImpl]
+.VotesImpl
+* xref:#VotesComponent-get_votes[`++get_votes(self, account)++`]
+* xref:#VotesComponent-get_past_votes[`++get_past_votes(self, account, timepoint)++`]
+* xref:#VotesComponent-get_past_total_supply[`++get_past_total_supply(self, timepoint)++`]
+* xref:#VotesComponent-delegates[`++delegates(self, account)++`]
+* xref:#VotesComponent-delegate[`++delegate(self, delegatee)++`]
+* xref:#VotesComponent-delegate_by_sig[`++delegate_by_sig(self, delegator, delegatee, nonce, expiry, signature)++`]
+--
+
+[.contract-index]
+.Internal implementations
+--
+.InternalImpl
+* xref:#VotesComponent-_delegate[`++_delegate(self, account, delegatee)++`]
+* xref:#VotesComponent-move_delegate_votes[`++move_delegate_votes(self, from, to, amount)++`]
+* xref:#VotesComponent-transfer_voting_units[`++transfer_voting_units(self, from, to, amount)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#VotesComponent-DelegateChanged[`++DelegateChanged(delegator, from_delegate, to_delegate)++`]
+* xref:#VotesComponent-DelegateVotesChanged[`++DelegateVotesChanged(delegate, previous_votes, new_votes)++`]
+--
+
+[#VotesComponent-VotingUnitsTrait-Implementations]
+==== ERC20VotesImpl
+
+[.contract-item]
+[[VotesComponent-ERC20VotesImpl-get_voting_units]]
```

Comment:
```suggestion
[#VotesComponent-ERC20VotesImpl]
==== ERC20VotesImpl

[.contract-item]
[[VotesComponent-ERC20VotesImpl-get_voting_units]]
```
There's no section for VotingUnitTrait-Implementations so I think we can remove that in favor of ERC20VotesImpl. This allows the `get_voting_units` ID to logically follow